DOCS-11123-ibmmq-xa-transactions-kt (#1503)
mulesoft/docs-connectors: ibm-mq/1.6/modules/ROOT/pages/ibm-mq-transactions.adoc

== XA Transactions

IBM MQ Connector supports a create and join XA transactions feature that enables the

> connector to participate in a transaction with multiple resources. This also enables to execute transactional actions with other connections of IBM MQ or other connectors such as Database Connector or VM Connector.

Reviewer: ```suggestion
connector to participate in a transaction with multiple resources. This also enables the connector to execute transactional actions with other connections of IBM MQ or other connectors such as Database Connector or VM Connector.
```

[NOTE]

> IBM MQ Connector does not support nesting XA transactions because of an IBM MQ driver limitation. When you work with nesting transactions, outer transactions get suspended and they resume after inner transactions finish. The failure occurs during the resumption attempt of outer transactions due to the driver implementation.

Reviewer: ```suggestion
IBM MQ Connector does not support nested XA transactions because of an IBM MQ driver limitation. When you work with nested transactions, outer transactions get suspended and then resume after the inner transactions finish. The failure occurs when the outer transactions are resumed, due to the driver implementation.
```
